lib/config/secrets.ts

```diff
@@ -85,21 +85,28 @@ function replaceSecretsInString(
 
 function replaceSecretsinObject(
   config_: RenovateConfig,
-  secrets: Record<string, string> = {}
+  secrets: Record<string, string> = {},
+  deleteSecrets = true
```

Comment:
```suggestion
  secrets: Record<string, string>,
  deleteSecrets
```

I think they are always passed?

Reply:
They are not, we have occurrences where we just pass the config and nothing else.
AFAIK for TS there is no way to use named parameters, so all optional parameters are required if we want to pass `deleteSecrets`
However, I can change all calls to use all parameters.